tests/pallas/pallas_test.py

```diff
@@ -587,7 +587,7 @@ def test_array_reduce(self, op, dtype, axis):
       out_dtype = jnp.int32
     def make_x(key):
       if jnp.issubdtype(dtype, jnp.integer):
-        return random.shuffle(key, jnp.arange(m * n, dtype=dtype)).reshape(m, n)
+        return random.permutation(key, jnp.arange(m * n, dtype=dtype), independent=True).reshape(m, n)
```

Comment:
Actually, could you format this so it is <=80 characters per line? Thanks.

Reply:
Not sure why `pre-commit` doesn't catch it, but running flake8 on its own does. Seems bugged tbh.

![image](https://github.com/google/jax/assets/9093549/ce85aa4c-35c1-43bb-83aa-7edee56d712f)
![image](https://github.com/google/jax/assets/9093549/660ee485-7e50-4f0f-9572-86d0e994871f)

Lots of other stuff is not caught by the pre-commit flake8 as well.

Reply:
Anw, just FYI, not sure how to fix.